web/packages/extension/src/background.ts

```diff
@@ -156,6 +157,7 @@ async function disableSWFTakeover() {
         await utils.declarativeNetRequest.updateDynamicRules({
             removeRuleIds: [1, 2, 3],
         });
+        utils.storage.sync.set({ responseHeadersUnsupported: false });
```

Comment:
I don't need this redundancy to be honest, but it's probably fine. Willing to hear other opinions though.

Honestly, I think disableSWFTakeover can run indefinitely and set nothing. https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/updateDynamicRules#removeruleids says any invalid IDs are ignored.